content/work/timedesigner.md

```diff
@@ -81,15 +114,403 @@ press:
+) }}
+
+For optimal flexibility,
+an experience may overlap in time with another experience.
+Thanks to CSS Grid,
+we are able to place these in a sensible way with minimal effort.
+Using `grid-auto-flow` with a value of `dense`,
+we can allow the row to place experiences where they
+fit, regardless of where they were added in the markup.
+Additionally, we can assign the row a minimum height
+and allow it to grow taller as needed using `minmax`.
+
+```css
+.row-grid {
+  grid-auto-flow: dense;
+  grid-auto-rows: minmax(1.75rem, auto);
+}
+```
+
+{{ embed.figure(
+  data=overlapexperience,
+  caption='CSS Grid allows us to backfill available
+    space of items regardless of document ordering.
+    This can lead to accessibility issues if not applied with care.'
+) }}
+
+### Drag-and-Drop Schedule Interface with CSS Grid
+
+This use of CSS grid combined beautifully
+with an intuitive drag-and-drop interface.
+An experience's `--start` variable is
+updated continuously while being dragged,
+allowing it to move responsively along the grid.
+
+{{ embed.figure(
+  data=[{
+    img: 'work/timedesigner/dragdrop.jpg',
+    alt: 'close up of an experience being dragged on the scheduling grid'
+  }],
+  caption='Each rectangle -- or "experience" --
+    can be dragged and dropped,
+    updating the CSS variables.'
+) }}
+
+### Print-to-PDF Reports
+
+Users needed the ability to view, print or download
+PDF reports of their scenarios
+using print-specific layouts.
+To do this, we created
+custom print styles,
+and used print-to-pdf
+in a [headless Chrome](https://github.com/fate0/pychrome) instance
+to render an HTML string to a PDF.
+
+{{ embed.figure(
+  data=[{
+    img: 'work/timedesigner/reports.jpg',
+    alt: 'preview of the schedule in a table view
+      filled with color coded experiences with times on top,
+      and categories on the left'
+  }],
+  caption='Viewable reports can be printed or downloaded via print-to-pdf.'
+) }}
+
+### Site-Wide Undo & Redo
+
+We also implemented a site-wide undo/redo feature
+(available via visible buttons
+or familiar `Ctrl-Z`/`Ctrl-Shift-Z`/`Ctrl-Y` keyboard shortcuts)
+through a custom action manager.
+Each action taken on the site
+is added to a stack,
+storing both "forward" and "backward" effects.
+Undoing an action
+triggers the "backward" effect
+of the most recent action,
+popping it off the end of the stack.
+Redoing an action runs the "forward" effect again,
+adding the action back to the end of the stack.
+
+{{ embed.figure(
+  data=[{
+    img: 'work/timedesigner/undo-redo.jpg',
+    alt: 'scenario with redo button highlighted'
+  }],
+  caption='Each action taken on the site is added to a stack,
+    storing both "forward" and "backward" effects.'
+) }}
+
+## On-Going Development
+
+{{ embed.figure(
+  data=[{
+    img: 'work/timedesigner/fullapp.jpg',
+    alt: 'scenario with multiple experiences selected'
+  }],
+  caption='Selecting a color for experiences in a TimeDesigner schedule.'
+) }}
```

Comment:
I believe this image shows the old design. If it is OK with you @SondraE I think this screenshot might be a good replacement? 

![td-group-edit-color](https://user-images.githubusercontent.com/1581694/132907603-5bb9613d-976a-4a3f-a538-5417209daa9b.jpg)